testsuite/devserver.go

```diff
@@ -158,6 +161,9 @@ func prepareCommand(options *DevServerOptions, host, port, namespace string) []s
 	if options.DBFilename != "" {
 		args = append(args, "--db-filename", options.DBFilename)
 	}
+	if options.UIPort != "" {
+		args = append(args, "--ui-port", options.UIPort)
```

Comment:
Does this also enable if not enabled? Docstring should mention that too

Reply:
No it does not enable the UI

Reply:
I am matching the behaviour of the dev server here

Reply:
I think in a lot of cases they are independently set via env vars and you may disable UI without changing port especially in hierarchical config cases like k8s/helm.